eZ/Bundle/EzPublishCoreBundle/Tests/SiteAccess/SiteAccessMatcherRegistryTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
```

Comment:
CS consistency: please keep here one blank line.
```suggestion

/**
```